backend/app/services/channels/telegram/handler.py

```
@@ -0,0 +1,626 @@
+            )
+
+        # Parse callback data
+        action, value = TelegramKeyboardBuilder.parse_callback_data(query.data or "")
+
+        if action == CallbackAction.CANCEL.value:
+            # Delete the keyboard message
+            try:
+                if query.message:
+                    await query.message.delete()
+            except Exception:
+                pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)